ui/component/publishForm/view.jsx

```diff
@@ -180,12 +252,38 @@ function PublishForm(props: Props) {
     }
   }, [modal]);
 
-  const isLivestream = mode === PUBLISH_MODES.LIVESTREAM;
+  // move this to lbryinc OR to a file under ui, and/or provide a standardized livestreaming config.
+  function checkLivestreams(channelId, signature, timestamp) {
+    // $FlowFixMe Bitwave's API can handle garbage
```

Comment:
this shouldn't need the `$FlowFixMe` comment and also remove the bitwave comment